README.md

```diff
@@ -455,31 +455,33 @@ Parameters controlling both management server creation and cluster creation:
+| ` `                              | `WORKER_MACHINE_GEN`                      | SCS        | `genw01`                                 | Suffix for worker node resources, to be changed for rolling upgrades                                                                                                                                       |
+| `anti_affinity`                  | `OPENSTACK_ANTI_AFFINITY`                 | SCS        | `true`                                   | Use anti-affinity server groups to prevent k8s nodes on same host (soft for workers, hard for controllers)                                                                                                 |
+| ` `                              | `OPENSTACK_SRVGRP_CONTROLLER`             | SCS        | `nonono`                                 | Autogenerated if `anti_affinity` is `true`, eliminated otherwise                                                                                                                                           |
+| ` `                              | `OPENSTACK_SRVGRP_WORKER`                 | SCS        | `nonono`                                 | Autogenerated if `anti_affinity` is `true`, eliminated otherwise                                                                                                                                           |
+| `deploy_occm`                    | `DEPLOY_OCCM`                             | SCS        | `true`                                   | Deploy the given version of OCCM into the cluster. `true` (default) chooses the latest version matching the k8s version. You can specify `master` to chose the upstream master branch. Don't disable this. |
+| `deploy_cindercsi`               | `DEPLOY_CINDERCSI`                        | SCS        | `true`                                   | Deploy the given (or latest matching for the default true value) of cinder CSI.                                                                                                                            |
+| `etcd_unsafe_fs`                 | `ETCD_UNSAFE_FS`                          | SCS        | `false`                                  | Use `barrier=0` for filesystem on control nodes to avoid storage latency. Use for multi-controller clusters on slow/networked storage, otherwise not recommended.                                          |
+| `testcluster_name`               | (cmd line)                                | SCS        | `testcluster`                            | Allows setting the default cluster name, created at bootstrap (if `controller_count` is larger than 0)                                                                                                     |
+| `restrict_kubeapi`               | `RESTRICT_KUBEAPI`                        | SCS        |  `[ ]`        
```

Comment:
Did the explanation for `restrict_kubeapi` get lost in the merge process?
Overall, I'm unhappy with the amount of whitespace changes that we recently have and which increases the risk for merge conflicts and thus for mistakes when resolving these ...

Reply:
So am I.... I will remove the branch and put cleaner code in a new one.